rpms/microshift.yml

```diff
@@ -0,0 +1,15 @@
+mode: wip
+arches:
+- x86_64
+- aarch64
+content:
+  source:
+    git:
+      branch:
+        target: main
+      url: git@github.com:openshift/microshift.git
+      web: https://github.com/openshift/microshift
+    specfile: packaging/rpm/microshift.spec
```

Comment:
What command it used to build the RPM? Are you running "make rpm" from the package, or something standard of your own based on the spec only